### DISCUSSION TITLE
Redis

### DISCUSSION THREAD

**Discussion by HosMercury** [maxcountryman/tower-sessions#136] [Q&A] [answered]
what redis crate do you use ?

**Comment by maxcountryman** [comment, marked as answer]
Any Redis client may be used as a store. Simply implement the SessionStore trait with your preferred client. 

Alternatively, we've also used fred to implement a Redis session store (with the same trait) that can be used as well.

Generally speaking, we very much encourage folks to implement their own session stores.
